heroku.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9
+
+# Install dependencies
+RUN apt-get update
+RUN apt-get install curl libpq-dev postgresql-client nodejs yarnpkg -y
+
+# Set working directory
+WORKDIR /home/indexer
+
+# Copy source code
+COPY . .
+
+# Install indexer
+RUN pip3 install poetry==1.6.1
+RUN poetry install
+RUN poetry export --without-hashes --format=requirements.txt > requirements.txt
```

Comment:
this is to work around the weird heroku bug i mentioned.
this file does not need to be tracked, since it's being generated by poetry.

Reply:
As mentioned above, maybe worth making note of this within the file.

Reply:
5fab0dbd67735c6a8fb7be2759278507d0522929